mep/books/views.py

```diff
@@ -57,9 +55,17 @@ def get_form(self, *args, **kwargs):
     solr_sort = {
         'relevance': '-score',
         'title': 'sort_title_isort',
+        'author': 'sort_authors_isort',
+        'pubdate': '-pub_date_i',
+        'circulation': '-event_count_i',
+        'circulation_date': 'first_event_date_i',
+        'pubdate': 'pub_date_i',
```

Comment:
codefactor's note about `pubdate` being listed twice seems to be correct. should we remove one?

Reply:
Whoops, holdover from when there were two versions. Thanks for flagging